src/Libraries/Microsoft.Extensions.Http.Diagnostics/Microsoft.Extensions.Http.Diagnostics.csproj

```diff
@@ -27,7 +27,7 @@
 
   <PropertyGroup>
     <Stage>normal</Stage>
-    <MinCodeCoverage>94</MinCodeCoverage>
+    <MinCodeCoverage>91</MinCodeCoverage>
```

Comment:
Code coverage has dropped because we had to replace code generated logger messages with their actual implementations (i.e. see [here](https://github.com/dotnet/extensions/pull/4545/files#diff-01befa772fdd79453ebf661760fa2509b4cfc9f0cbe25c7644cc1680c8678ef1R129)) because of the following limitation of the `LoggerMessage` #4576. As soon as we implement #4576, we increase the code coverage again.